composer.json

```diff
@@ -4,25 +4,40 @@
     "type": "drupal-profile",
     "license": "GPL-2.0-or-later",
     "homepage": "https://github.com/localgovdrupal/localgov",
-    "repositories": [
-        {
+    "repositories": {
+        "dropzone": {
+            "type": "package",
+            "package": {
+                "name": "enyo/dropzone",
+                "version": "6.0.0-beta.2",
+                "type": "drupal-library",
+                "dist": {
+                    "url": "https://github.com/dropzone/dropzone/releases/download/v6.0.0-beta.2/dist.zip",
+                    "type": "zip"
+                }
+            }
+        },
```

Comment:
Not so sure about this, is this fixing the version that will get installed? It means that it will need to be updated each time. Maybe we need to look at the [composer merge plugin](https://packagist.org/packages/wikimedia/composer-merge-plugin) that some modules like [webform](https://git.drupalcode.org/project/webform/-/blob/6.2.x/composer.libraries.json?ref_type=heads)  or maybe use [asset packagist](https://asset-packagist.org/) instead?

Reply:
It's fixing the version, yes, but there's hasn't been a new version in about 2 years so we probably won't be under too much pressure.

Reply:
Hi Andy, any further thoughts on this, or are we okay to approve it?